hack/update-roles-in-csv.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash -e
+
+#
+# Copyright 2019 IBM Corp.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+get_latest_csi_version (){
+  latest_csi_version=$(cat version/version.go | grep -i driverversion | awk -F = '{print $2}')
+  echo ${latest_csi_version//\"}
+}
+
+declare -a bundle_names=(
+  "ibm-block-csi-operator-community"
+  "ibm-block-csi-operator"
```

Comment:
we [already maintain the bundle names](https://github.com/IBM/ibm-block-csi-operator/blob/3c8d55c9c198782e604c81194574a406d26e47a0/build/ci/olm_validation.sh#L37-L38), please move them to a common place

Reply:
done

Reply:
I didn't mean that [wildcard](https://github.com/IBM/ibm-block-csi-operator/blob/b8ab411cecc9c6a4945b6b330bdc5288a1beb9c5/hack/update-yamls-with-the-same-content.sh#L33) is better. I meant that we should maintain explicit "consts"/list of the bundle names/paths in one common place